dataland-frontend/src/services/ApiClients.ts

```diff
@@ -46,8 +115,8 @@ export class ApiClientProvider {
     return new constructor(configuration, "/documents");
```

Comment:
Any API-Client using the getConstructedApi<T>() or getConstructedDocumentManager<T> functions will not use the Axios interceptor. This should be avoided.

Reply:
Done.

Reply:
Can we have a quick call about this? I've not yet understood the changes make here

Reply:
Clarified in a call